climada/entity/exposures/litpop/litpop.py

```diff
@@ -992,47 +1017,49 @@ def gridpoints_core_calc(data_arrays, offsets=None, exponents=None,
 
     Returns
     -------
-    result_array : np.array of same shape as arrays in data_arrays
+    result : np.array of same shape as arrays in data_arrays
```

Comment:
This is really nitpicking, but `result` is a terrible variable name ^^.

Reply:
Also fine. Although `data` is not a better name than `result`.

Reply:
Then I'll do that. My goal is not to refactor this function 😅

Reply:
Done in 620d3460f9fc4e159fd2ced3aecda705e5bd2fb6